<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav)"/>

  <group ns="$(arg uav_name)">
    
    <node name="automatic_start" pkg="mrs_general" type="automatic_start_mbzirc" output="screen" >
      
      <rosparam file="$(find mrs_general)/config/automatic_start_mbzirc.yaml" />

      <!-- Subscribers -->
      <remap from="~mavros_state" to="mavros/state" />

      <!-- Service callers -->
      <remap from="~change_state" to="mrs_communication/change_state" />
    
    </node>
  
  </group>

</launch>
